BossMod/Components/InterceptTether.cs

```diff
@@ -0,0 +1,62 @@
+﻿namespace BossMod.Components;
+
+// generic component for tethers that need to be intercepted eg. to prevent a boss from gaining buffs - (Ported From BMR)
+public class InterceptTether(BossModule module, ActionID aid, uint tetherID) : CastCounter(module, aid)
```

Comment:
How is it different from existing tether components (tankbustertether / baitawaytether)? No aoe, can target both other enemies and players, anything else?

Reply:
It is for tethers that simply need to be intercepted to prevent something (eg. boss gaining buffs)

Reply:
> How is it different from existing tether components (tankbustertether / baitawaytether)? No aoe, can target both other enemies and players, anything else?

I tried using tankbustertether for this mechanic but couldn't get it to work. I'm still a newbie with vbm ;P

Reply:
removed this module